download_openapi.sh

```
@@ -0,0 +1,7 @@
+set -a
+source .env
+set +a
+if [ -z ${SWAGGER_CONFIG_PATH+x} ];
+then echo "SWAGGER_CONFIG_PATH is unset";
```

Should we throw an error here?

---

I think it's okay. If it's not set, the local `openapi.yaml` will be used.